biomage/utils/data.py

```diff
@@ -130,8 +260,45 @@ def copy_dynamodb_records(
             click.echo(f"Failed inserting records: {e}")
 
 
+def copy_project_record(
+    sandbox_id, staging_experiments, source_table, target_table, config
+):
+
+    dynamodb = boto3.client("dynamodb")
+    for experiment_id in staging_experiments:
+        project_id = dynamodb.get_item(
+            TableName="experiments-production",
+            Key={"experimentId": {"S": experiment_id}}
+        ).get("Item")['projectId']['S']
+
+        item = dynamodb.get_item(
+            TableName=source_table,
+            Key={"projectUuid": {'S' : project_id}}
+        ).get("Item")
+
+        try:
+            dynamodb.put_item(
+                TableName=target_table,
+                Item={
+                    **item,
+                    "projectUuid": {
+                        "S": f"{sandbox_id}-{project_id}"
+                    },
+                    **modified_records(
+                        item,
+                        target_table,
+                        config,
+                        sandbox_id=sandbox_id
+                    ),
+                },
+            )
+        except Exception as e:
```

Comment:
as discussed, do not catch general exceptions, if this fails it should really fail and then, given a specific exception either solve the issue (still without catching) or add then a specific exception catch if it's something that might happen sometimes (like a network error, etc..)

Reply:
Hmm.. there's nothing specific that I'd want to catch here, that's why I'm using a catchall. By saying "it should really fail" do you mean that it should stop the next steps in the process?

Reply:
If you are not expecting an error then I don't think there should be any try at all. If, for example, some error like "incorrect table"  or "invalid UUID" fails then yes, this should just raise an exception so that the person using it can fix it instead of hiding it (even if there's a message somewhere in the logs). The thing is, the command is used to copy data, if you **can't** copy the data, then you should just raise the exception and fail. The result won't be correct otherwise. Remember the pragmatic advice, **crash, don't trash**

Reply:
Yea, I'd want the program to crash if it can't copy the project records. I'm removing the try-catch.